netlify/edge-functions/workspaces-card.tsx

```diff
@@ -0,0 +1,282 @@
+                  }}
+                >
+                  <img
+                    style={{
+                      width: 25.2,
+                      height: 25.2,
+                    }}
+                    src={getOrgUsernameAvatar(orgUsername)}
+                  />
+                  <span>{repoName}</span>
+                </li>
+              );
+            })}
+            <li>{restOfReposCount > 0 ? `+${restOfReposCount}` : ""}</li>
+          </ul>
+        </div>
+        <div
+          style={{
+            marginTop: "113.5px",
+            fontSize: "84px",
+            fontWeight: 700,
+            letterSpacing: "-2px",
+            lineHeight: "101.66px",
+          }}
+        >
+          {workspaceName}
+        </div>
+        <p style={{ marginTop: "11.5px", fontSize: "32px", fontWeight: 400 }}>{workspaceDescription}</p>
+        <p style={{ marginTop: "94px", fontSize: "26px", fontWeight: 500 }}>Past {range} days</p>
+        <div
+          style={{
+            marginTop: "8px",
+            display: "flex",
+            alignItems: "baseline",
+            justifyContent: "space-between",
+          }}
+        >
+          <ul
+            style={{
+              display: "flex",
+              listStyle: "none",
+              alignItems: "baseline",
+            }}
+          >
+            <li style={statContainerStyles}>
+              <img style={statIconStyles} src={`${new URL(`/assets/og-images/workspaces/git-merge.png`, req.url)}`} />
+              <span style={statTextContainerStyles}>
+                <span style={statValueContainerStyles}>
+                  <span style={statValueStyles}>{repoStats.pull_requests.merged}</span>
+                </span>
+                <span style={statSubTextStyles}>Merged PRs</span>
+              </span>
+            </li>
+            <li style={statContainerStyles}>
+              <img
+                style={statIconStyles}
+                src={`${new URL(`/assets/og-images/workspaces/issue-closed.png`, req.url)}`}
+              />
+              <span style={statTextContainerStyles}>
+                <span style={statValueContainerStyles}>
+                  <span style={statValueStyles}>{repoStats.issues.closed}</span>
+                </span>
+                <span style={statSubTextStyles}>Closed Issues</span>
+              </span>
+            </li>
+            <li style={statContainerStyles}>
+              <img
+                style={{ ...statIconStyles, width: 33.3 }}
+                src={`${new URL(`/assets/og-images/workspaces/star.png`, req.url)}`}
+              />
+              <span style={statTextContainerStyles}>
+                <span style={statValueContainerStyles}>
+                  <span style={statValueStyles}>{repoStats.repos.stars}</span>
+                </span>
+                <span style={statSubTextStyles}>Stars</span>
+              </span>
+            </li>
+          </ul>
+        </div>
+        <img
```

Comment:
This is absolute because when it was in the flexbox container, although it was aligned on the baseline, visually it didn't look good. As discussed with @isabensusan, I moved it down slightly.